### DISCUSSION TITLE
Timed out retrying after 4000ms: cy.type() failed because the center of this element is hidden from view:

### DISCUSSION THREAD

**Discussion by AsIncube** [cypress-io/cypress#15584] [Questions and Help]
Timed out retrying after 4000ms: cy.type() failed because the center of this element is hidden from view: how can i solve this??

**Comment by Amritanshu-Amoga** [comment]
Hello sir, How have get over this issue

**Comment by ZhiduReborn** [comment]
Facing the same issue... We have a popup window on our website and it can be closed when I manually click anywhere on the page. But in the Cypress test, no matter where I click, it doesn't close, even this `cy.get("body").click("bottomRight", {force: true})`. Without force, it returned this error.
